modules/multi-arch-uninstalling-using-cli.adoc

```diff
@@ -0,0 +1,75 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-uninstalling-using-cli_{context}"]
+
+= Uninstalling the Multiarch Tuning Operator by using the CLI
+
+You can uninstall the Multiarch Tuning Operator by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` permissions.
+* You have installed the {oc-first}.
+* You have deleted the `ClusterPodPlacementConfig` custom resource (CR).
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Get the `currentCSV` value for the Multiarch Tuning Operator by running the following command:
++
+[source,terminal]
+----
+$ oc get subscription.operators.coreos.com multiarch-tuning-operator -n <namespace> -o yaml | grep currentCSV
+----
++
+.Example output
+[source,terminal]
+----
+currentCSV: multiarch-tuning-operator.v0.9.0
+----
+
+. Delete the subscription by running the following command:
++
+[source,terminal]
+----
+$ oc delete subscription.operators.coreos.com openshift-multiarch-tuning-operator -n <namespace>
+----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com "openshift-multiarch-tuning-operator" deleted
+----
+
+. Delete the CSV for the Multiarch Tuning Operator in the target namespace using the `currentCSV` value by running the following command:
++
+[source,terminal]
+----
+$ oc delete clusterserviceversion <currentCSV_value> -n <namespace> <1>
+----
+<1> Replace `<currentCSV>` with the `currentCSV` value for the Multiarch Tuning Operator. For example: `multiarch-tuning-operator.v0.9.0`
++
+.Example output
+[source,terminal]
+----
+clusterserviceversion.operators.coreos.com "multiarch-tuning-operator.v0.9.0" deleted
+----
+
+.Verification
+
+* To verify that the Multiarch Tuning Operator is uninstalled, run the following command:
++
+[source,terminal]
+----
+$ oc get csv -n <namespace>
```

Comment:
```suggestion
$ oc get csv -n <namespace>
```
If namespace is a user-replaceable value, I would footnote it and say so.

Reply:
Updated as per the suggestion.